### PR TITLE
LPS-173367 | Automation test Creation, Alerts are not styled correctly

### PR DESCRIPTION
**Pull request by andreluis2022**
𝙏𝙞𝙘𝙠𝙚𝙩:
https://issues.liferay.com/browse/LPS-170570

𝘿𝙚𝙨𝙘ription:
Alerts are not styled correctly

ClayAlertFeedback#IconCanDisplayTooltip

### CODE REVIEW

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
@mateusralv

**Comment by mateusralv** [comment]
Hey @andreluis2022 the SF still failing because you forgot to add the "LPS-173367" in the commit (https://github.com/liferay-frontend/liferay-portal/pull/3044/commits/3e1ad1a7da319930d9dbb52f9cb9c7ca36858bf0) title.

**Comment by andreluis2022** [comment]
> 

true, I'll fix it

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
Hey @andreluis2022 Could take a look at this sf issue? Thanks 😄

**Comment by andreluis2022** [comment]
> Ei@andreluis2022Poderia dar uma olhada neste problema sf? Obrigadosorriso

yes

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
cc @matuzalemsteles can you verify the clay usage here is right?

**Comment by bryceosterhaus** [comment]
LGTM! just need to resolve merge conflict

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:frontend-clay

**Comment by mateusralv** [comment]
The test is passing in CI 🎉

 [LocalFile.ClayAlertFeedback#IconCanDisplayTooltip](https://testray.liferay.com/home/-/testray/case_results/2198583963?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2198579604%26testrayRunId%3D2198583673%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertFeedback.testcase [hunk 1]
```diff
@@ -38,16 +38,33 @@ definition {
...
+				layoutName = "Clay Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "Clay Page",
+				widgetName = "Clay Sample");
 
-		// TODO LPS-133020 IconCanDisplayTooltip pending implementation
+			Navigator.gotoPage(pageName = "Clay Page");
+		}
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertFeedback.testcase:59]: Hey @andreluis2022 could you remove this part? These steps are being covered in the set up

**Review by mateusralv — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertFeedback.testcase (hunk 1 above)
Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertFeedback.testcase]: Done!

**Comment by mateusralv** [comment]
ci:test:frontend-clay

**Review by mateusralv — APPROVED** [review]

**Comment by mateusralv** [comment]
ci:forward

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**